apps/website/src/pages/collaborations.tsx

```diff
@@ -348,10 +365,10 @@ const CollaborationsPage: NextPage = () => {
               together to learn about the importance of conservation.
             </p>
           </div>
-
           <Creators className="mt-6" />
         </Section>
       </div>
+      <SubNav links={sectionLinks} />
```

Comment:
```suggestion

      <SubNav links={sectionLinks} />
```
Nit: newlines around sections in the page